cmd/tap/README.md

```diff
@@ -0,0 +1,179 @@
+- `TAP_CURSOR_SAVE_INTERVAL`: how often to save cursor (default: `5s`, set to `0` to disable)
+- `TAP_REPO_FETCH_TIMEOUT`: timeout for fetching repo CARs from PDS (Go duration syntax, e.g. `180s`)
+- `TAP_FULL_NETWORK_MODE`: track all repos on the network (default: `false`)
+- `TAP_SIGNAL_COLLECTION`: track all repos with at least one record in this collection (e.g. `app.bsky.actor.profile`)
+- `TAP_COLLECTION_FILTERS`: comma-separated collection filters, wildcards accepted (e.g., `app.bsky.feed.post,app.bsky.graph.*`)
+- `TAP_DISABLE_ACKS`: fire-and-forget mode, no client acks (default: `false`)
+- `TAP_WEBHOOK_URL`: webhook URL for event delivery (disables WebSocket mode)
+- `TAP_OUTBOX_ONLY`: run in outbox-only mode (no firehose, resync, or enumeration) (default: `false`)
+- `TAP_LOG_LEVEL`: log verbosity (`debug`, `info`, `warn`, `error`, default: `info`)
+
+## Delivery Modes
+
+Tap supports three delivery modes:
+
+**WebSocket with acks** (default): Client sends acks each event once it has been processed/persisted. Ensures that no data is lost and client does not need to handle cursors. It's recommended to use a client library such as (@TODO) when using this mode.
+
+**Fire-and-forget**: Set `TAP_DISABLE_ACKS=true`. Events are sent and considered "acked" once the client receives them. Simpler but may result in data loss. Recommended for testing purposes or when data integrity is not critical.
+
+**Webhook**: Set `TAP_WEBHOOK_URL=http://...`. Events are POSTed as JSON. Events considered "acked" once the webhook responds with a 200. Recommended for lower throughput serverless environments.
+
+
+## Network Boundary Modes
+
+Tap syncs a subset of repos in the network. It can operate in three modes for determining this network boundary.
+
+**Dynamically Configured** (default):  Tap starts out tracking no repos. Specific repos can then by added via `/add-repos` and removed via `/remove-repos`.
+
+**Collection Signal**: Set `TAP_SIGNAL_COLLECTION=com.example.nsid`. Track all repos that have at least one record in the specified collection. Many applications create a "declaration" or "profile" in a repo when that repo uses that application
+
+**Full Network**: Set `TAP_FULL_NETWORK_MODE=true`. Enumerates and tracks all findable repos on the entire network. Resource-intensive and takes days/weeks to complete backfill.
+
+## Event Format
+
+Events are delivered as JSON:
+
+**Record events** (create, update, delete):
+
+```json
+{
+  "id": 12345,
+  "type": "record",
+  "record": {
+    "did": "did:plc:abc123",
+    "collection": "app.bsky.feed.post",
+    "rkey": "3kb3fge5lm32x",
+    "action": "create",
+    "cid": "bafyreig...",
+    "record": {
+      "text": "Hello world!",
+      "$type": "app.bsky.feed.post",
+      "createdAt": "2024-10-07T12:00:00.000Z"
+    },
+    "live": true
+  }
+}
+```
+
+**User events** (handle or status changes):
+
+```json
+{
+  "id": 12346,
+  "type": "user",
```

Comment:
maybe `account` instead of `user`? I don't have a super strong opinion, but we generally use "account" instead of "user" elsewhere in the protocol and SDKs

Reply:
I used `user` specifically because this combines both `identity` and `account` semantics.

But maybe it's better to just use an existing term. I think i'd prefer `identity` over `account`